assets/js/opensuse-theme.js

```diff
@@ -133,8 +133,9 @@ function osMoreInformation(os) {
 
   moreInfoOpened = true
   //get the height of the main container
-  containerHeight = $('#opensuse-os').outerHeight()
-
+  containerHeight = $('#opensuse-os').outerHeight();
+  window.location.hash = '0';
```

Comment:
I am adding hashtag, because in Chrome when you hit Backspace button it navigate you to previous page.